apps/remix-ide/src/app/providers/injected-skale-chaos-testnet-provider.tsx

```diff
@@ -0,0 +1,18 @@
+import * as packageJson from '../../../../../package.json'
+import { InjectedCustomProvider } from './injected-custom-provider'
+
+const profile = {
+  name: 'injected-skale-chaos-testnet-provider',
+  displayName: 'Injected SKALE Chaos Testnet',
+  kind: 'provider',
+  description: 'Injected SKALE Chaos Testnet Provider',
+  methods: ['sendAsync', 'init'],
+  version: packageJson.version
+}
+
+export class InjectedSKALEChaosTestnetProvider extends InjectedCustomProvider {
+
+  constructor () {
+    super(profile, 'SKALE Chaos Testnet', '0x50877ed6', ['https://staging-v3.skalenodes.com/v1/staging-fast-active-bellatrix'])
```

Comment:
With this details, Metamask shows this while adding the network:
`The submitted currency symbol does not match what we expect for this chain ID.` 

as currency is set as `UNKNOWN` on metamask. Can we add a value for that too?

Reply:
Please set currency symbol to sFUEL. 

For clarity it has 18 decimals. 

Thank you